packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -447,7 +449,9 @@ export class MgtFileList extends MgtTemplatedComponent {
             `
           )}
         </ul>
-        ${!this.hideMoreFilesButton && this.pageIterator && this.pageIterator.hasNext
+        ${!this.hideMoreFilesButton &&
+        this.pageIterator &&
+        (this.pageIterator.hasNext || this._files.length > this.pageSize)
```

Comment:
This is causing the more files button to not show up when there are more files to load but they are less than the page size. 

```suggestion
        (this.pageIterator.hasNext || this._files.length)
```

Reply:
It actually has to be like this `this._preloadedFiles.length - this.files.length > 0`